src/Presentation/Presenters/MembershipApplicationConfirmationHtmlPresenter.php

```diff
@@ -74,4 +75,14 @@ private function getBankDataArguments( PaymentMethod $payment ): array {
 		return [];
 	}
 
+	/**
+	 * Maps the membership application's status to a translatable message key
+	 *
+	 * @param bool $isConfirmed
+	 * @return string
+	 */
+	private function mapStatus( bool $isConfirmed ): string {
```

Comment:
Small thing: I'd probably have passed the whole application here. Now the caller kinda knows that only the isConfirmed part is being used to determine the status